noxfile.py

```diff
@@ -147,6 +147,21 @@ def snippets(session):
             session.env['GCLOUD_ORGANIZATION'] = '1081635000895'
         else:
             session.skip('Credentials must be set via environment variable.')
+    if not os.environ.get('GCLOUD_PROJECT', ''):
```

Comment:
Hiya! 

Re: adding the pub/sub dependency

1. Create a `requirements.txt` file in the `docs` directory.

```
google-cloud-pubsub==1.3.0
```

2. At the top of the snippets session, add the following to install dependencies from the requirements.txt:
```python
session.install("-r", "docs/requirements.txt")